packages/mui-codemod/src/v5.0.0/base-use-named-exports.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Finds the last segment of the path starting with @mui/base.
+ *
+ * @example @mui/base/Menu ➔ Menu
+ * @example @mui/base/utils/Foo ➔ Foo
+ */
+function getBaseImportIdentifier(path) {
+  const source = path?.node?.source?.value;
+  if (!source) {
+    return null;
+  }
+
+  const baseImportPathMatch = source.match(/@mui\/base\/(?:(?:.*)\/)*([^/]+)/);
```

Comment:
Can we simplify the regexp? We don't imports nested more than three times. Or is there other use-case we are trying to capture with it?